src/commands.ts

```diff
@@ -46,61 +61,53 @@ export function registerCommands(ext: Extension) {
 // Restarts the language server, updating the client in the extension object.
 const restartServer = (ext: Extension) => async () => {
   await ext.api.client.stop();
-  ext.api = new LanguageServerAPI(ext.ctx, ext.config);
+  const activeIndex = ext.config.activeAccount
+  const {name, address} = activeIndex != null ? ext.config.accounts[activeIndex] : {name: "", address: ""}
+  ext.api = new LanguageServerAPI(ext.ctx, ext.config, ext.emulatorState, {name, address});
```

Comment:
Can you use `getActiveAccount()` here?

```suggestion
  const account = ext.config.getActiveAccount()
  ext.api = new LanguageServerAPI(ext.ctx, ext.config, ext.emulatorState, account);
```